config/jest/vite.config.js

```diff
@@ -0,0 +1,33 @@
+module.exports = {
+    rootDir: "../../",
+    transform: {
+        "^.+\\.jsx?$": "<rootDir>/config/jest/vite.transform.js",
+    },
+    testMatch: ["<rootDir>/vite-tests/*.test.js"],
+    testEnvironment: "@khanacademy/jest-environment-vite",
+    testEnvironmentOptions: {
+        capabilities: {
+            browserName: "chrome",
+            chromeOptions: {
+                args: ["headless", "disable-gpu"],
+            },
```

Comment:
We'll want to make this configurable so that we can both run these tests on more browser on circleci as well as run them in a non-headless browser locally which is useful for debugging things.  Ideally though I'd like to figure out how to forward console.log, console.error, and exception stack traces back from the browser back to jest and report the errors there.